cards.js

```diff
@@ -16,8 +16,42 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
+function buildDeck(suits, values){
+  // make a new deck so that each suit gets one of each card value
+  var deck = [];
+  for(suit in suits) {
+    for(value in values){
+      deck.push({suit: suits[suit], value: values[value]});
+    }
+  }
+return deck;
+}
+
+var newDeck = buildDeck(suits, values)
+var shuffledDeck = shuffle(newDeck)
 
-// Put some code in here!
+console.log(shuffledDeck)
 
+function highCard(deck, values) {
+  // make the players and give them each a card; player1 gets the first card in the deck array and player 2 gets the second card in the deck array
+  var player1 = deck[0];
+  var player2 = deck[1];
+  console.log("Player 1's card is: " + player1.value + " of " + player1.suit);
+  console.log("Player 2's card is: " + player2.value + " of " + player2.suit);
+  
+  currentGame = [player1, player2];
+
+  if(values.indexOf(player1.value) > values.indexOf(player2.value)) {
+    console.log("Player 1 wins!");
+  }
+  else if(values.indexOf(player1.value) < values.indexOf(player2.value)) {
+    console.log("Player 2 wins!");
+  }
+  else {
+    console.log("It's a tie!");
+  }
```

Comment:
Looks good.  2 suggestions:
1. It seems strange to pass the values array AND the deck to highCard.  Can we store each card's rank within the card?
2. I would like to see the scoring/comparison concept represented clearer.  Maybe...
```js
function compareCards(card1_rank, card2_rank){
  if(card1_rank > card2_rank) {
    return 1;
  } else if(card1_rank < card2_rank){
    return -1;
  } else {
   return 0;
  }
}

switch (compareCards(player1_card.rank, player2_card.rank) {
    case 1:
        console.log("Player 1 wins!");
        break; 
    case -1: 
        console.log("Player 2 wins!");
        break;
    case 0:
        console.log("It's a tie!");
    end
});
```
Note: written from memory.  May have minor syntax issues.